test/playback.test.js

```diff
@@ -243,6 +249,7 @@ QUnit.test('loops', function(assert) {
       player.vhs.mediaSource.addEventListener('sourceopen', () => {
         assert.ok(true, 'sourceopen triggered after ending stream');
         done();
+        player.loop(false);
```

Comment:
is this necessary?

Reply:
no, its a holdover from something I was testing.